app/js/components/rule/ruleGroupCard/ruleGroupCard.jade

```diff
@@ -22,7 +22,7 @@
       i.fa.fa-arrow-right
     card
       card-header
-        .md-card-header-title(translate) {{index + 1}} of {{plugin.rules.length}}: {{rule.description}}
+        .md-card-header-title(translate) {{index + 1}} of {{:: plugin.rules.length}}: {{rule.description}}
         span.label.label-default.pull-right.text-uppercase(translate, ng-show='rule.acked') inactive
         .md-card-header-cta(ng-if='config.isPlannerEnabled')
           incident-icon(rule-id='rule.rule_id')
```

Comment:
all-severity-icons on line 34 is broken (the values never change). Ideally, it should be enough to swap the rule and the bound components would update automatically. This seems to have been broken even before.

Reply:
Can you go into more detail? I am not sure how to repro the breakage

![1](https://user-images.githubusercontent.com/129737/35110254-57eefc22-fc46-11e7-8fd2-83c9003b9c63.png)

![2](https://user-images.githubusercontent.com/129737/35110253-57e29cb6-fc46-11e7-835e-39abda5f5915.png)

Reply:
![screenshot_20180118_195552](https://user-images.githubusercontent.com/167352/35115805-ac80e550-fc89-11e7-98d9-efaa002165c1.png)

:confused:

Reply:
@iphands oh wait! So it works fine when you use the blue arrows to switch between cards but it does not work when I use actual physical keyboard :arrow_left: :arrow_right: keys for switching

Reply:
that bugs predates your changes. I guess it's because `$document.on` handler is called outside of angular digest cycle. I pushed a fix for that one